multihost_test/bz_automation/test_automation.py

```diff
@@ -252,3 +252,54 @@ def test_libpam_raise_line_buffer_size_limit(self, multihost, create_localusers,
         client.run_command("su - local_anuj -c exit")
+
+    def test_pam_access(self, multihost, create_localusers, bkp_pam_config):
+        """
+        :title: Using "pam_access", ssh login fails with this entry in
+                /etc/security/access.conf "+:username:127.0.0.1"
+        :id: d2a52d84-9fe0-11ef-a76a-3c18a0580700
+        :bugzilla: https://issues.redhat.com/browse/RHEL-65223
+        :steps:
+            1. Enable "with-pamaccess" feature using authselect
+            2. Configure  /etc/security/access.conf
+            3. Try to log in with the user
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. Should succeed
+        """
+        client = multihost.client[0]
+        file_location = "/multihost_test/bz_automation/script/2014458.sh"
+        multihost.client[0].transport.put_file(os.getcwd() + file_location, '/tmp/2014458.sh')
+        client.run_command("authselect select sssd --force")
+        client.run_command("authselect enable-feature with-pamaccess")
+        assert "with-pamaccess" in client.run_command("authselect current").stdout_text
+        client.run_command(f"echo '+:local_anuj:127.0.0.1' >> /etc/security/access.conf")
+        client.run_command("echo '-:ALL:ALL' >> /etc/security/access.conf")
+        client.run_command("sh /tmp/2014458.sh")
+
+    def test_pam_access_account(self, multihost, create_localusers, bkp_pam_config):
+        """
+        :title: pam_access(sshd:account): cannot resolve hostname "LOCAL"
+                after upgrading to pam-1.5.1-19
+        :id: fcbae668-9fe0-11ef-9fbb-3c18a0580700
+        :bugzilla: https://issues.redhat.com/browse/RHEL-39943
+        :steps:
+            1. Enable pam_access with policy line(+:bob:LOCAL) in /etc/security/access.conf
```

Comment:
You are missing the setup phase, including some steps that you mentioned in the previous test case that are missing here.